### PR TITLE
homework_week1 added

### PR DESCRIPTION
**Pull request by amjadiftikhar**

### CODE REVIEW

**Comment by amjadiftikhar** [comment]
Hi Jawad

Thank you for the feedback. Actually I have fixed the bug and would like to request you, if you get a chance to get through. Many thanks once again.

File: nodejs/week1/mealProject/src/backend/index.js [hunk 1]
```diff
@@ -1,93 +1,112 @@
...
+// 	next();
+// });
+
+const mealsRoute = require('./routes/mealsRoute');
+const reservationsRoute = require('./routes/reservationsRoute');
+// const randomReservationRoute = require('./routes/randomReservationRoute');
 
 // Respond with the json for all the meals (including it's reviews) that are cheap (you define what a cheap meal is)
 
-app.get("/meals", (req, res, next) => {
```
Comment (jawkha) [inline nodejs/week1/mealProject/src/backend/index.js]: This should have been:
```js
app.get("/cheap-meals", (req, res, next) => {
```

File: nodejs/week1/mealProject/src/backend/index.js [hunk 2]
```diff
@@ -1,93 +1,112 @@
...
+// app.get("/meals", (req, res, next) => {
+// 	const cheapMeal = meals.filter(mealPrice => mealPrice.price < 100);
+// 	// console.log(cheapMeal);
+// 	res.json(cheapMeal);
+// 	next();
+// });
 
 //Respond with the json for all the meals (including it's reviews) that can fit lots of people
 
-app.get("/meals", (req, res, next) => {
```
Comment (jawkha) [inline nodejs/week1/mealProject/src/backend/index.js]: This should have been:
```js
app.get("/large-meals", (req, res, next) => {
```

File: nodejs/week1/mealProject/src/backend/index.js [hunk 3]
```diff
@@ -1,93 +1,112 @@
...
+// 		}
+// 	});
+// 	// console.log(maxMeal);
+// 	res.json(maxMeal);
+// 	next();
+// });
 
 //Respond with the json for a random meal (including it's reviews)
 
-app.get("/meals", (req, res, next) => {
```
Comment (jawkha) [inline nodejs/week1/mealProject/src/backend/index.js]: This should have been:
```js
app.get("/meal", (req, res, next) => {
```

File: nodejs/week1/mealProject/src/backend/index.js [hunk 4]
```diff
@@ -1,93 +1,112 @@
...
-});
+// app.get("/reservations", (req, res, next) => {
+// 	res.json(reservations);
+// 	// console.log(reservations);
+// 	next();
+// });
 
 //Respond with the json for a random reservation
 
-app.get("/reservations", (req, res, next) => {
```
Comment (jawkha) [inline nodejs/week1/mealProject/src/backend/index.js]: This should have been:
```js
app.get("/reservation", (req, res, next) => {
```

**Comment by amjadiftikhar** [comment]
Hi Jawad

Thank you for the feedback. Actually I got the point but I just tried to do it with the router. Anyway I have fixed it now and I am thankful to you once again.

Regards

**Pull request merged**